### DISCUSSION TITLE
Translatable fields not converted

### DISCUSSION THREAD

**Discussion by lemoinet** [pods-framework/pods#6593] [Ideas]
My translatable fields are not converted to their corresponding fields when I add a translation to a custom post type.
By this, it also displays an error message that the required field is not filled in while it has a value.

**Comment by JoryHogeveen** [comment]
Hi lemoniet,

Can you give me a bit more info about the fields? By default when you create a translation all fields need to be refilled with the right content for that language. I know Polylang has the sync option, but this doesn't work for all fields.

**Comment by lemoinet** [comment]
I have a custom required field "Project Type" (Field Type: Relationship) linked to a multilingual custom taxonomy.
when i want to attach a new translation to a newly created custom post item, it is not converting the values from one translation to the other translation, making the required field throw an error (required must not be empty) and clearing my other multilingual fields (with field type relationship)

**Comment by JoryHogeveen** [comment]
I assume you have the sync option enabled at Polylang?
Anyhow.. relationship field types don't support syncing since a "sync" feature only copies the data from the database value (and thus the incorrect values when using multilingual content).

Unfortunately there is no workaroud currently since there are some limitations:
- When adding a new draft, it's "challenging" to hook into here and prefill the values.
- When the translation doesn't exists there are two options: 
1. **remove the relationship** (and causing the sync option to fail when it syncs again after you save) 
2. **auto-add the translation** (and causing an empty post to be created from the original, which in turn also triggers the sync function that will remove data from the already existing relationships since it's an empty post).

Long story short. Untill there is a solution for this (I am busy with it..) you'll be better off disabling the sync feature for metadata/custom fields.

Edit: The same applies to the `file` fields (images, audio, etc) when you have them enabled for translation.

**Comment by JoryHogeveen** [comment]
Related: https://github.com/pods-framework/pods/issues/1851

**Comment by JoryHogeveen** [comment]
@sc0ttkclark I'd like your thoughts on the above: https://github.com/pods-framework/pods/issues/3773#issuecomment-244914400
